Checkout/Sample/Source/Card Tokenisation/Card Details/ExpirationDatePicker.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  ExpirationDatePicker.swift
+//
+//
+//  Created by Harry Brown on 08/02/2022.
+//
+
+import UIKit
+
+class ExpirationDatePicker: UIPickerView {
+  private static var secondsInYear: TimeInterval = 31536000
```

Comment:
When doing something like this it may be easier to add the complete source of the number:
```suggestion
private static var secondsInYear: TimeInterval = 60 * 60 * 24 * 365
```